pkg/webhook/certs.go

```diff
@@ -496,3 +510,27 @@ func (r *ReconcileVWH) Reconcile(request reconcile.Request) (reconcile.Result, e
 
 	return reconcile.Result{}, nil
 }
+
+// ensureCertsExist ensure the cert files exist.
+func (cr *CertRotator) ensureCertsExist() {
```

Comment:
can we add a test for this?

Reply:
Thanks for the review! 

This function is an IPC from cert generation to adding controllers (specifically webhooks) to the controller manager on the fly after it's started. It checks the file existence in the mountPath. 

I am not sure how to test `os.Stat(certFile)`. What I did is a manual test in a deployment. Please let me know if it's good enough or do you have any suggestions on the testing?

Reply:
is it possible to do something like this https://talks.golang.org/2012/10things.slide#8 to mock the filesystem?

Reply:
Is there a specific case you're interested in unit testing? Right now we have two logic paths:

File exists -> close happy channel
File missing -> close sad channel

Reply:
sure, if its not worth the effort we can skip the unit test

Reply:
Hi @sozercan , do you mean it's okay to skip this unit test? If so, could you please review the change and merge it if it looks fine for you? Thanks!

Reply:
@yiqigao217 approved, can you rebase and then we can merge. thanks!

Reply:
Sure thing! Done and the pretests are passed! Thanks!